frontend/src/App.tsx

```diff
@@ -278,6 +291,12 @@ class App extends PureComponent<Props, State> {
    */
 
   handleInitialize(initializeMsg: Initialize): void {
+    if (!this.isServerVersionUpdated(initializeMsg)) {
```

Comment:
if wasn't updated, will reload? 
it's rigth the `!`?

Reply:
Yes, if it is not updated means that the versions are different so we have to reload it